src/components/MatchItem/Avatars.tsx

```diff
@@ -0,0 +1,73 @@
+import { AvatarGroup, Chip, Avatar, Badge } from '@mui/material'
+import { Participant } from '../../types/Match'
+
+export const Avatars = ({ participants }: { participants: Participant[] }) => {
+  const participantsByScore = participants.sort((a, b) => b.score - a.score)
+
+  const participant = participantsByScore.map((participant) => {
+    if (participant.isWinner) {
+      return (
+        <>
+          <Chip
+            sx={{ marginTop: '2px', marginRight: '2px' }}
+            avatar={
+              <Badge
+                anchorOrigin={{
+                  vertical: 'top',
+                  horizontal: 'right',
+                }}
+              >
+                <Avatar
+                  alt={participant.fullName}
+                  src="1.jpg"
```

Comment:
Is this right?

Reply:
No, I have checked the documentation, and it should work like this:

If there is an error loading the avatar image, the component falls back to an alternative in the following order:
   - the provided children
   - the first letter of the alt text
   - a generic avatar icon

With this, I decided to provide the first letter as a child to be able to pass an empty string to src.